admin/tabs/activity/tracking/ActivityTrackingModel.ts

```diff
@@ -330,12 +329,21 @@ export class ActivityTrackingModel extends HoistModel {
     }
 
     private async loadLookupsAsync() {
-        const lookups = await XH.fetchJson({url: 'trackLogAdmin/lookups'});
-
-        this.filterChooserModel.fieldSpecs.forEach(spec => {
-            const {field} = spec;
-            if (lookups[field]) spec.values = lookups[field];
-        });
+        try {
+            const lookups = await XH.fetchJson({url: 'trackLogAdmin/lookups'});
+            this.filterChooserModel.fieldSpecs.forEach(spec => {
+                const {field} = spec,
+                    lookup = lookups[field] ? compact(lookups[field]) : null;
+
+                if (!isEmpty(lookup)) {
+                    spec.values = lookup;
+                    spec.enableValues = true;
+                    spec.hasExplicitValues = true;
```

Comment:
Great that we wait until we've confirmed we have non-null values before setting these flags 👍